instance-scheduler/rds.go

```diff
@@ -23,99 +24,38 @@ type IRDSInstancesAPI interface {
 	DescribeDBInstances(ctx context.Context, params *rds.DescribeDBInstancesInput, optFns ...func(*rds.Options)) (*rds.DescribeDBInstancesOutput, error)
 }
 
-func StopStartTestRDSInstancesInMemberAccount(rdsClient IRDSInstancesAPI, action string) *RDSInstanceCount {
-	action = strings.ToLower(action)
-	rdscount := &RDSInstanceCount{RDSActedUpon: 0, RDSSkipped: 0}
+func StopStartTestRDSInstancesInMemberAccount(RDSClient IRDSInstancesAPI, action string) *RDSInstanceCount {
```

Comment:
Should this function be renamed given that it no longer includes the stop action?